### ISSUE TITLE
NOT ABLE TO IMPORT MY SUBSCRIPTIONS FROM YOUTUBE

### ISSUE THREAD

**Issue by lightningzword** [SkyTubeTeam/SkyTube#1046]
I know that you guys have used NewPipe's API for the backend of the app but this bug is in the NewPipe app too. There too, I am unable to add my subscriptions as it only excepts the ".json" file format. Please look into this ASAP as it is very inconvenient for users. 
Hope to see this fixed in the next update :)

**Comment by LordJon** [comment]
The new way of importing should be possible to implement by using the Google takeout export method. It generates a csv containing all subscriptions (unselect all other YouTube data when exporting).

I discovered SkyTube today in my search for an alternative app in my journey to (partially) DeGoogle. Would love to import my subscription list.

**Comment by ToXinE** [comment]
when I try to use takout importing feature it says quota has been exceeded.

**Comment by C-O-D** [comment]
SkyTube Extra 2.981 still doesn't import abo.csv!
I mean, that's such an important feature, I don't get it...

**Comment by LordJon** [comment]
Yeah I think the devs need some more time as Google completely changed the export format. For now I stopped using the app as manually adding 400 subscriptions isn't realistic.

I'll check this app later again, it had great potential. The add blocking isn't necessary for me though, but I think this is also part of the privacy aspect?

**Comment by tkalfaoglu** [comment]
When I try to import a csv the app just closes

**Comment by LordJon** [comment]
There are multiple reports about this problem which is probably why this one got closed.
Tkalfaoglu, the reason why the app closes is because this problem hasn't been fixed in the public release yet (no clue if there is a fix at all). Google changed the format so importing the csv doesn't work atm.

**Comment by RedSnt** [comment]
Nothing new to add except that I'm really sad this "import youtube subscriptions" feature is broken at the moment, because SkyTube with its SponsorBlock support, looks like the best of adfree youtube players available.

**Comment by JamesCullum** [comment]
Issue is caused by NewPipe and has been fixed upstream: https://github.com/TeamNewPipe/NewPipe/issues/7411
